docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -27,7 +27,11 @@ Use [phpinfo()](http://php.net/manual/en/function.phpinfo.php) to inspect your c
  * PostgreSQL ([third party module](https://addons.silverstripe.org/add-ons/silverstripe/postgresql), community supported)
  * SQL Server ([third party module](https://addons.silverstripe.org/add-ons/silverstripe/mssql), community supported)
  * SQLite ([third party module](https://addons.silverstripe.org/add-ons/silverstripe/sqlite3), community supported)
- 
+
+ ### Connection mode (sql_mode) when using MySQL server >=5.7.5
+
+ MySQL versions >=5.7.5 the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY` setting. In most cases it is recommended to leave this setting as-is because it results in deterministic SQL. However for some advanced cases the sql_mode can be configured on the database connection via the configuration API.
```

Comment:
```suggestion
In MySQL versions >=5.7.5, the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY` setting. It is generally recommended to leave this setting as-is because it results in deterministic SQL. However, for some advanced cases, the sql_mode can be configured on the database connection via the configuration API (see `MySQLDatabase::$sql_mode` for more details.) This setting is only available in Silverstripe CMS 4.7 and later.
```